src/Config.hs

```diff
@@ -340,9 +340,9 @@ checkPathExistsAndIsEmpty path = do
 
 -- * motd
 
+-- | Log the message (motto) of the day (like /etc/motd).
 logmotd :: Config -> FilePath -> IO ()
```

Comment:
config is only part of it, and `logConfig` is much less informative than `motd` (well ok, at least if you know unix long enough for this to be a word.)  i want to keep the name.